tests/unit/lib/path_util_test.py

```diff
@@ -129,6 +129,9 @@ def test_single_file(self):
         )
         self.assertEqual(linked_bundle_path.is_archive, True)
         self.assertEqual(linked_bundle_path.is_archive_dir, False)
+        self.assertEqual(
```

Comment:
do we also need to check for the existence of the archive?

Reply:
The archive won't exist here as we haven't uploaded anything yet. This is just testing the parsing logic of `parse_linked_bundle_url`